sdv/single_table/copulas.py

```diff
@@ -216,6 +216,9 @@ def get_learned_distributions(self):
                 "Distributions have not been learned yet. Please fit your model first using 'fit'."
             )
 
+        if self._model is None:
+            return {}
```

Comment:
I think that this is the more future proof version, reason being that if we update `self._model` to be initialized earlier, but `to_dict` is empty, we will return the empty `{}` otherwise it will crash when accessing `parameters['columns']` on an empty dict:
````suggestion
        if not hasattr(self._model, 'to_dict') or not self._model.to_dict():
            return {}